docs/reference/commandline/run.md

```diff
@@ -198,8 +198,13 @@ The `-w` lets the command being executed inside directory given, here
     $ docker run -it --storage-opt size=120G fedora /bin/bash
 
 This (size) will allow to set the container rootfs size to 120G at creation time.
-User cannot pass a size less than the Default BaseFS Size. This option is only
```

Comment:
According to the Doc, I am a little confused.
For xfs + overlay2 quota control, whether container rootfs size is limited to 120G, or the top layer is limited to 120G?